client/styles/modal.scss

```diff
@@ -69,9 +71,39 @@
             padding-bottom: 16px;
         }
     }
+    .compact-modal-body th{
+        font-style: normal;
+        font-weight: bold;
+        font-size: 16px;
+        line-height: 120%;
+        text-align: left !important;
+        font-family: 'Inter', sans-serif !important;
+
+    }
+    .compact-modal-body td{
+        font-family: 'Inter', sans-serif !important;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 16px;
+        line-height: 120%;
+        letter-spacing: -0.2px;
+    }
     .compact-modal-actions {
         padding: 6px 28px 28px;
     }
+
+// avatar div
+.user-avatar{
+    width: 25px !important;
+    height: 25px !important;
+    padding-right: 14px !important;
+  }
+  
+  .Jdenticon{
+    width: 25px !important;
+    height: 25px !important;
+  }
+
```

Comment:
Same as above. The modal component should have its own class. This prevents unexpected interactions with our other components.

Reply:
Hope this is fine : https://github.com/hicommonwealth/commonwealth/commit/07ccd5304836b74e0f071341f6d6e5028949db43